flowable-xml-parser/src/main/kotlin/com/valb3r/bpmn/intellij/plugin/flowable/parser/nodes/process/nested/definitions/ErrorEventDefinition.kt

```diff
@@ -3,5 +3,8 @@ package com.valb3r.bpmn.intellij.plugin.flowable.parser.nodes.process.nested.def
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
 
 data class ErrorEventDefinition(
-    @JacksonXmlProperty(isAttribute = true) val errorRef: String? = null
+    @JacksonXmlProperty(isAttribute = true) val errorRef: String? = null,
+    @JacksonXmlProperty(isAttribute = true, namespace = "http://flowable.org/bpmn") val errorVariableName: String? = null,
+    @JacksonXmlProperty(isAttribute = true, namespace = "http://flowable.org/bpmn") val errorVariableLocalScope: Boolean? = null,
+    @JacksonXmlProperty(isAttribute = true, namespace = "http://flowable.org/bpmn") val errorVariableTransient: Boolean? = null
```

Comment:
`"http://flowable.org/bpmn"` - namespace is typically defined in parser, no need to define it here